ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1686,28 +1686,9 @@ AP_Rally *GCS_MAVLINK_Plane::get_rally() const
  */
 bool GCS_MAVLINK_Plane::set_mode(const uint8_t mode)
```

Comment:
It's odd that we send a `uint32_t` down to GCS_MAVLINK but take a `uint8_t` back up.  All vehicles will probably do this ATM.

Reply:
works as-is